lib/Page.js

```diff
@@ -782,17 +782,7 @@ class Page extends EventEmitter {
    * @return {!Promise<!Array<string>>}
    */
   async select(selector, ...values) {
-    return await this.$eval(selector, (element, values) => {
-      if (element.nodeName.toLowerCase() !== 'select')
-        throw new Error('Element is not a <select> element.');
-      const options = Array.from(element.options);
-      element.value = undefined;
-      for (const option of options)
-        option.selected = values.includes(option.value);
-      element.dispatchEvent(new Event('input', { 'bubbles': true }));
-      element.dispatchEvent(new Event('change', { 'bubbles': true }));
-      return options.filter(option => option.selected).map(option => option.value);
-    }, values);
+    return this.mainFrame().select(selector, ...values);
```

Comment:
can you please state in `page.select` documentation that it's a shortcut for `page.mainFrame().select`.

E.g. see how it's done for [page.$](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageselector)